src/lib/Microsoft.Health.Events/EventHubProcessor/ResumableAssignedPartitionProcessor.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Events.EventHubProcessor
+{
+    public class ResumableAssignedPartitionProcessor : IResumableEventProcessor
+    {
+        private PartitionLockingBackgroundService _lockingBackgroundService;
+
+        private bool _isRunning = false;
+
+        public ResumableAssignedPartitionProcessor(PartitionLockingBackgroundService lockingBackgroundService)
+        {
+            _lockingBackgroundService = lockingBackgroundService;
+        }
+
+        public void Dispose()
+        {
+            // todo?
```

Comment:
After further investigation, we don't appear to be registering the PartitionLockingBackgroundService as a hosted service, so we should definitely ensure it is properly stopped on dispose (though we will likely need to handle an exception for the service already being exposed in case it is disposed first).  We don't need to dispose since the DI should handle that for us.